seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/BaseSourceConfigOptions.java

```diff
@@ -40,6 +41,24 @@ public class BaseSourceConfigOptions {
                     .noDefaultValue()
                     .withDescription("The file path of source files");
 
+    public static final Option<FilePathRule> FILE_PATH_RULE =
```

Comment:
add `SinkFactory#optionRule()`?

Reply:
sorry,I should create a darft pr, these issues will be fixed later